bundle/deploy/terraform/import.go

```diff
@@ -0,0 +1,111 @@
+package terraform
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/hashicorp/terraform-exec/tfexec"
+)
+
+type BindOptions struct {
+	AutoApprove  bool
+	ResourceType string
+	ResourceKey  string
+	ResourceId   string
+}
+
+type importResource struct {
+	opts *BindOptions
+}
+
+// Apply implements bundle.Mutator.
+func (m *importResource) Apply(ctx context.Context, b *bundle.Bundle) error {
+	dir, err := Dir(ctx, b)
+	if err != nil {
+		return err
+	}
+
+	// If the bundle.tf.json file does not exist, write it.
+	// This is necessary because the import operation requires the resource to be defined in the Terraform configuration.
+	_, err = os.Stat(filepath.Join(dir, "bundle.tf.json"))
+	if err != nil {
+		err = bundle.Apply(ctx, b, bundle.Seq(Interpolate(), Write()))
+		if err != nil {
+			return fmt.Errorf("terraform write: %w", err)
+		}
+	}
```

Comment:
Why only if it doesn't exist? It could exist but be stale.

Reply:
Could you comment on this one?

Reply:
Stale would mean different from local changes in bundle config, correct? My idea is not to include any local changes and only limit to TF import ones so if there’s already remote state which we pull from remote, just use it

Reply:
Yes, correct. If I have a deployment already (thus a `bundle.tf.json` file), and I run `generate job` followed by an import, the local `bundle.tf.json` file won't include the definition for that job yet, and it will fail (I think)?